package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "default",
+  "version": "1.0.0",
+  "description": "Onboarding project",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/jquery": "^3.3.29"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc -w --project ./"
```

Comment:
I'm guessing TypeScript was installed as a global node package on your PC. If you want to install it locally (to this project), run `npm install --save-dev typescript`. This will add typescript to the dependencies of this project, making the version of typescript explicit for any other devs that may work on it. Like we spoke earlier, this is probably not something that is that important to a small on-boarding project like this, but on anything bigger it is good to lock this down.